manage_user_update.php

```diff
@@ -186,6 +199,10 @@
 		$t_changes .= lang_get( 'username_label' ) . ' ' . $t_old_username . ' => ' . $f_username . "\n";
 	}
 
+	if( $t_password_changed ) {
+		$t_changes .= lang_get( 'password' ) . ' ' . $t_old_password . ' => ' . "[REDACTED]" . "\n";
```

Comment:
`[REDACTED]` is not localized, and we shouldn't show old password.  I would just have a localized message here that password was changed.